pytential/symbolic/mappers.py

```diff
@@ -600,31 +596,31 @@ def map_ones(self, expr, enclosing_prec):
         return "Ones[%s]" % stringify_where(expr.dofdesc)
 
     def map_inverse(self, expr, enclosing_prec):
-        return "Solve(%s = %s {%s})" % (
+        return "Solve({} = {} {{{}}})".format(
```

Comment:
Meh.

Reply:
d9f8f97

Reply:
Oh no, I missed that! They changed my python into lisp!